### ISSUE TITLE
nav-underline with disabled button is not grayed

### ISSUE THREAD

**Issue by Flachdachs** [twbs/bootstrap#38765]
### Prerequisites

- [X] I have [searched](https://github.com/twbs/bootstrap/issues?utf8=%E2%9C%93&q=is%3Aissue) for duplicate or closed issues
- [X] I have [validated](https://html5.validator.nu/) any HTML to avoid common problems
- [X] I have read the [contributing guidelines](https://github.com/twbs/bootstrap/blob/main/.github/CONTRIBUTING.md)

### Describe the issue

When using nav-underline as described in section [Underline](https://getbootstrap.com/docs/5.3/components/navs-tabs/#underline) of "Navs & tabs" but using a `button` element instead of an `a`nchor a disabled button still has the primary color.

### Reduced test cases

```html
<ul class="nav nav-underline">
  <li class="nav-item">
    <button class="nav-link" disabled>Disabled</button>
  </li>
</ul>
```

### What operating system(s) are you seeing the problem on?

Windows

### What browser(s) are you seeing the problem on?

Chrome, Firefox

### What version of Bootstrap are you using?

5.3.0

**Comment by Flachdachs** [comment]
Can't create a live demo without creating an account there. Closing this issue due to formal error. Bug still exists.

**Comment by mdo** [comment]
We have this as a class now to your point, but not the pseudo state. https://getbootstrap.com/docs/5.3/components/navs-tabs/#underline

Re-opening so we can tackle.

**Comment by Flachdachs** [comment]
As far as I understand the [button documentation for disabled state](https://getbootstrap.com/docs/5.3/components/buttons/#disabled-state) the class `disabled` is only a workaround for `a`nchors, because they ignore the `disabled` attribute. So `button`s should be fine without the class. It's also written there that `aria-disabled="true"` should be set for these disabled anchors, which is also not the case for the examples in both Navs & tabs and Navbar.

**Comment by julien-deramond** [comment]
@patrickhlauke I've tried something to tackle the initial issue in https://github.com/twbs/bootstrap/pull/38774. I'd like your thoughts on the `aria-disabled="true"` mentioned in the previous comment in order to see if I need to update my PR or create another one to tackle this topic separately.

**Comment by patrickhlauke** [comment]
@julien-deramond sorry, coming in late, will have a look at this soon

**Comment by patrickhlauke** [comment]
for the `aria-disabled` aspect, we should follow-through in our examples with what we say in the actual documentation, so yes we'd need a PR that backfills this to our docs examples

**Comment by mdo** [comment]
Fixed by #38774
